### DISCUSSION TITLE
Electron + NextJS... Why?

### DISCUSSION THREAD

**Discussion by daveteu** [saltyshiomix/nextron#144] [Q&A]
I've using Nextjs for most of my projects, mainly due to SSR and I can just use /api/ routing, with Nextjs being a very complete boiler plate.

However, in desktop app, i assume we don't need SSR, and obviously /api routes are hosted else where, may I know why are you using NextJs instead of just CRA?

**Comment by lacymorrow** [comment]
In a word: Prototyping.

In 11 words: faster prototyping, simpler routing, and IMO a nicer API

But really it comes down to personal preference :)

**Reply by daveteu** [reply]
Thank you, it works well so far, until I tried putting in my current project folder in and then I start getting all the pre-rendering errors.

Anything to note when we package our app in electron?

E.g. (does server side rendering work? does getstaticprops etc need to be removed?)

**Reply by daveteu** [reply]
The error I'm referring to is this. It works fine in my project but when I put in electron folder this comes up and the errors are all not helpful.

```
[=   ] info  - Generating static pages (0/3)
Error occurred prerendering page "/home". Read more: https://err.sh/next.js/prerender-error
Error: Minified React error #143; visit https://reactjs.org/docs/error-decoder.html?invariant=143 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
```
